src/lib/pickles/composition_types/composition_types.ml

```diff
@@ -227,6 +227,27 @@ module Wrap = struct
                 with type field = f ) ~dummy_scalar ~dummy_scalar_challenge
               ~challenge ~scalar_challenge ~bool ~feature_flags
               (fp : (fp, _, f) Snarky_backendless.Typ.t) =
+            let uses_lookup =
+              let { Plonk_types.Features.range_check0
+                  ; range_check1
+                  ; foreign_field_add = _ (* Doesn't use lookup *)
+                  ; foreign_field_mul
+                  ; xor
+                  ; rot
+                  ; lookup
+                  ; runtime_tables = _ (* Fixme *)
+                  } =
+                feature_flags
+              in
+              Array.reduce_exn ~f:Opt.Flag.( ||| )
+                [| range_check0
+                 ; range_check1
+                 ; foreign_field_mul
+                 ; xor
+                 ; rot
+                 ; lookup
```

Comment:
Is this one meant to represent the `Lookup` gate only? If so, then I guess `berkeley` should have it in `plonk_constraint_system.ml`, which right now only lives in `develop`.